.clang-format-ignore

```diff
@@ -0,0 +1,3 @@
+src/battle/battle_controller_player.c
```

Comment:
before anything else, is there a reason why these are ignored?

Reply:
bad formatting results specifically on these files

Reply:
what happened with it?

Reply:
i don't remember and didn't take notes

Reply:
ok I can probably look at it a bit myself and see

Reply:
just remembered what it was -- the NONMATCHINGS mess with things badly